OpenImageIO produces link errors

Linking the current version of OpenImageIO [2.3.10.1] produces link errors.

```

1>   Creating library C:\Users\Roger\source\repos\Projects\ImageViewers\Tinta\x64\Debug\QtOIIOPlugin.lib and object C:\Users\Roger\source\repos\Projects\ImageViewers\Tinta\x64\Debug\QtOIIOPlugin.exp
1>QtOIIOHandler.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl OpenImageIO_v2_3::TypeDesc::TypeDesc(enum OpenImageIO_v2_3::TypeDesc::BASETYPE,enum OpenImageIO_v2_3::TypeDesc::AGGREGATE,enum OpenImageIO_v2_3::TypeDesc::VECSEMANTICS,int)" (__imp_??0TypeDesc@OpenImageIO_v2_3@@QEAA@W4BASETYPE@01@W4AGGREGATE@01@W4VECSEMANTICS@01@H@Z) referenced in function "public: virtual bool __cdecl QtOIIOHandler::read(class QImage *)" (?read@QtOIIOHandler@@UEAA_NPEAVQImage@@@Z)
1>QtOIIOHandler.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl OpenImageIO_v2_3::TypeDesc::TypeDesc(struct OpenImageIO_v2_3::TypeDesc const &)" (__imp_??0TypeDesc@OpenImageIO_v2_3@@QEAA@AEBU01@@Z) referenced in function "public: virtual bool __cdecl QtOIIOHandler::read(class QImage *)" (?read@QtOIIOHandler@@UEAA_NPEAVQImage@@@Z)
1>QtOIIOPlugin.obj : error LNK2001: unresolved external symbol "__declspec(dllimport) public: __cdecl OpenImageIO_v2_3::TypeDesc::TypeDesc(struct OpenImageIO_v2_3::TypeDesc const &)" (__imp_??0TypeDesc@OpenImageIO_v2_3@@QEAA@AEBU01@@Z)
1>QtOIIOHandler.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: char const * __cdecl OpenImageIO_v2_3::TypeDesc::c_str(void)const " (__imp_?c_str@TypeDesc@OpenImageIO_v2_3@@QEBAPEBDXZ) referenced in function "public: virtual bool __cdecl QtOIIOHandler::read(class QImage *)" (?read@QtOIIOHandler@@UEAA_NPEAVQImage@@@Z)
1>QtOIIOHandler.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: bool __cdecl OpenImageIO_v2_3::TypeDesc::is_array(void)const " (__imp_?is_array@TypeDesc@OpenImageIO_v2_3@@QEBA_NXZ) referenced in function "bool __cdecl OpenImageIO_v2_3::operator!=(struct OpenImageIO_v2_3::TypeDesc const &,enum OpenImageIO_v2_3::TypeDesc::BASETYPE)" (??9OpenImageIO_v2_3@@YA_NAEBUTypeDesc@0@W4BASETYPE@10@@Z)
1>QtOIIOPlugin.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl OpenImageIO_v2_3::ustring::ustring(void)" (__imp_??0ustring@OpenImageIO_v2_3@@QEAA@XZ) referenced in function "bool __cdecl OpenImageIO_v2_3::getattribute(class OpenImageIO_v2_3::string_view,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &)" (?getattribute@OpenImageIO_v2_3@@YA_NVstring_view@1@AEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z)
1>QtOIIOPlugin.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: __cdecl OpenImageIO_v2_3::ustring::~ustring(void)" (__imp_??1ustring@OpenImageIO_v2_3@@QEAA@XZ) referenced in function "bool __cdecl OpenImageIO_v2_3::getattribute(class OpenImageIO_v2_3::string_view,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &)" (?getattribute@OpenImageIO_v2_3@@YA_NVstring_view@1@AEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z)
1>QtOIIOPlugin.obj : error LNK2019: unresolved external symbol "__declspec(dllimport) public: class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const & __cdecl OpenImageIO_v2_3::ustring::string(void)const " (__imp_?string@ustring@OpenImageIO_v2_3@@QEBAAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@XZ) referenced in function "bool __cdecl OpenImageIO_v2_3::getattribute(class OpenImageIO_v2_3::string_view,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > &)" (?getattribute@OpenImageIO_v2_3@@YA_NVstring_view@1@AEAV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z)
1>C:\Users\Roger\source\repos\Projects\ImageViewers\Tinta\x64\Debug\QtOIIOPlugin.dll : fatal error LNK1120: 7 unresolved externals
```

Windows. x64, Visual Studio 2019
Installed these projects:

vcpkg install openimageio[libraw,OpenJpeg,GIF]  --triplet x64-windows --recurse
vcpkg install opencv  --triplet x64-windows --recurse
vcpkg install tbb  --triplet x64-windows --recurse
vcpkg install lz4  --triplet x64-windows --recurse
vcpkg install dlib  --triplet x64-windows --recurse

--- a/ports/openimageio/portfile.cmake
+++ b/ports/openimageio/portfile.cmake
@@ -68,6 +68,7 @@ vcpkg_cmake_configure(
         "-DREQUIRED_DEPS=fmt;JPEG;PNG;Robinmap"
     MAYBE_UNUSED_VARIABLES
         ENABLE_INSTALL_testtex
+        ENABLE_IV
 )
 
 vcpkg_cmake_install()
@@ -88,12 +89,16 @@ if("viewer" IN_LIST FEATURES)
     )
 endif()
 
-# Clean
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/share/doc"
                     "${CURRENT_PACKAGES_DIR}/debug/include"
                     "${CURRENT_PACKAGES_DIR}/debug/share")
 
 vcpkg_fixup_pkgconfig()
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+    vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/OpenImageIO/export.h" "ifdef OIIO_STATIC_DEFINE" "if 1")
+endif()
+
 
 file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE.md")

--- a/ports/openimageio/vcpkg.json
+++ b/ports/openimageio/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "openimageio",
   "version": "2.5.12.0",
+  "port-version": 1,
   "description": "A library for reading and writing images, and a bunch of related classes, utilities, and application.",
   "homepage": "https://github.com/OpenImageIO/oiio",
   "license": "BSD-3-Clause",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6490,7 +6490,7 @@
     },
     "openimageio": {
       "baseline": "2.5.12.0",
-      "port-version": 0
+      "port-version": 1
     },
     "openjpeg": {
       "baseline": "2.5.2",

--- a/versions/o-/openimageio.json
+++ b/versions/o-/openimageio.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "ce78ee5a41eda148167cad88a553e111b40d4840",
+      "version": "2.5.12.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "8722c349a4a0df02d8003750f58868305250eddb",
       "version": "2.5.12.0",
       "port-version": 0

